msteams-platform/toolkit/Microsoft-365-Agents-Toolkit-CLI.md

```diff
@@ -0,0 +1,517 @@
+
+### `atk collaborator status` parameters
+
+The following table lists the parameters available for `atk collaborator status`:
+
+| Parameter                      | Required | Description                                                                                               |
+| :----------------------------- | :------- | :-------------------------------------------------------------------------------------------------------- |
+| `--env`                        | No       | Specifies the environment name for the project scaffolded by Microsoft 365 Agents Toolkit.                |
+| `--manifest-file`              | No       | Specifies the app manifest file path. The default value is `./appPackage/manifest.json`.                  |
+| `--entra-app-manifest-file -a` | No       | Select the path for the Entra manifest template file. The default value is `${folder}/aad.manifest.json`. |
+| `--all -a`                     | No       | Display all collaborators.                                                                                |
+| `--folder -f`                  | No       | Select the project's root folder. The default value is `./`.                                              |
+
+### `atk collaborator grant` parameters
+
+The following table lists the parameters available for `atk collaborator grant`:
+
+| Parameter                      | Required | Description                                                                                          |
+| :----------------------------- | :------- | :--------------------------------------------------------------------------------------------------- |
+| `--email`                      | No       | Enter the collaborator's email address.                                                              |
+| `--manifest-file`              | No       | Specifies the app manifest file path. The default value is `./appPackage/manifest.json`.             |
+| `--entra-app-manifest-file -a` | No       | Select the path for the Entra app manifest file. The default value is `${folder}/aad.manifest.json`. |
+| `--env`                        | No       | Specifies the environment name for the project scaffolded by Microsoft 365 Agents Toolkit.           |
+| `--folder -f`                  | No       | Select the project's root folder. The default value is `./`.                                         |
+
+### `atk collaborator` scenarios
+
+Grant permission for another Microsoft 365 account to collaborate on the app.
+
+```typescript
+atk collaborator grant -i false --manifest-file ./appPackage/manifest.json --env dev --email other@email.com
+```
+
+## `atk uninstall`
+
+Clean up resources associated with manifest ID, title ID, or an environment in Microsoft 365 Agents Toolkit generated project. Resources include app registration in Teams Developer Portal, bot registration in Bot Framework Portal, and uploaded custom apps in Microsoft 365 apps.
```

Comment:
Should it be Developer Portal rather than Teams Developer Portal?

Reply:
updated